MacrobenchmarkSample/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright 2021 The Android Open Source Project
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="androidx.benchmark.macro.target">
+
+    <application
+        android:allowBackup="false"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_notice"
+        android:requestLegacyExternalStorage="true"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.Macrobenchmarktarget">
+
+        <profileable android:shell="true" />
+
+        <!--
+        Activities need to be exported so the Macro Benchmarks can discover them
+        under the new package visibility changes for Android 11.
+        -->
+
+        <activity
+            android:name=".TrivialStartupActivity"
+            android:exported="true">
+
+            <intent-filter>
```

Comment:
remove this intent filter? want to make this minimal

Reply:
If we remove this, then we lose the ability to launch the app from the launcher. Do you really want to do that ?

Reply:
we can leave it in. I wasn't thinking people would launch the app from the launcher, but they might if they want to try capturing perf information manually to compare.